doc/howtos/using_ubuntu_as_sos.rst

```diff
@@ -0,0 +1,184 @@
+Using Ubuntu as the Service OS with the ACRN hypervisor
+
+
+Based on the ACRN open source project and getting started guide. This documentation is to guide people how to boot Ubuntu system as the 
+Service OS with the ACRN hypervisor. By default, in the getting started guide documentation of the ACRN open source project, Clear Linux
+is being used as the Service OS and User OS on top of the ACRN hypervisor. 
+Here we are trying to boot Ubuntu system as the Service OS and launch Clear Linux as User OS with the ACRN hypervisor on official 
+supported NUC, playing steps as below:
```

Comment:
Can we be more specific and indicate the model (using a link). I would like that we add a Baby Canyon (KBL NUC) to the list soon and at that point, the "official supported NUC" becomes ambiguous.

The instructions below would likely need to be tweaked a little for a different NUC.